cdxev/__main__.py

```diff
@@ -712,6 +714,50 @@ def create_build_public_bom_parser(
     return parser
 
 
+def create_list_command_parser(
+    subparsers: argparse._SubParsersAction,
+) -> argparse.ArgumentParser:
+    parser = subparsers.add_parser(
+        "list",
+        help=(
+            "Lists specific contents of the SBOM."
+            "Currently supported are the listing of license information and component information"
+        ),
+        usage=(
+            "cdx-ev list [-h] [--format {txt,csv}] [--skip-metadata] "
+            "[--output <file>] <operation> input"
+        ),
```

Comment:
as you have not done any specific changes to the usage, this is not really required

Reply:
Added {licenses, components}